test/validation/api/timer/timer.c

```diff
@@ -253,29 +257,29 @@ check_plain_queue_support(void)
 
 static int check_periodic_support(void)
 {
-	if (global_mem->no_periodic)
-		return ODP_TEST_INACTIVE;
+	if (global_mem->periodic_support)
+		return ODP_TEST_ACTIVE;
 
-	return ODP_TEST_ACTIVE;
+	return ODP_TEST_INACTIVE;
 }
 
 static int check_periodic_sched_support(void)
 {
-	if (global_mem->periodic_support[0] && global_mem->param.queue_type_sched)
+	if (global_mem->periodic_support && global_mem->param.queue_type_sched)
 		return ODP_TEST_ACTIVE;
 
 	return ODP_TEST_INACTIVE;
 }
 
 static int check_periodic_plain_support(void)
 {
-	if (global_mem->periodic_support[0] && global_mem->param.queue_type_plain)
+	if (global_mem->periodic_support && global_mem->param.queue_type_plain)
 		return ODP_TEST_ACTIVE;
 
 	return ODP_TEST_INACTIVE;
 }
 
-static void timer_test_capa_run(odp_timer_clk_src_t clk_src)
+static void timer_test_capa(void)
```

Comment:
It easier to read the code when all globals are passed though the same global mem variable. It this case:

clk_src =globals->clk_src;